### ISSUE TITLE
Bravo! Belle initiative!!

### ISSUE THREAD

**Issue by ClaudeCoulombe** [RobertGodin/JavaPasAPas#1]
Salut Robert et Daniel!

J'ai vu l'annonce de votre livre « Java pas à pas  » sur mon fil LinkedIn!
Super année 2024! 

Claude Coulombe

**Comment by RobertGodin** [comment]
Salut Claude ! Ancien voisin de bureau à l'U deM  :-) Te souviens-tu !
On a un livre analogue avec Python qui s'en vient sous peu :-) Et un autre en apprentissage machine pas mal avancé. Je suis tombé sur ton cours en vision en regardant ce qui a été fait dans le domaine en français et c'est du beau boulot ! 
Bonne année à toi aussi.

Robert Godin

**Comment by ClaudeCoulombe** [comment]
> Salut Claude ! Ancien voisin de bureau à l'U deM :-) Te souviens-tu ! 

Oui, dans le U ou le V 500 avec Michel Décary mon futur partenaire dans Machina Sapiens inc, Clément Lacaille, André Foisy, Denis Fortin, Michel Maksud, Louis-Philippe Demers, Philippe Bergeron et d'autres... On a souvent sauvé le monde! Sur une note plus triste, cela me rappelle aussi Daniel Langlois, décédé tragiquement en Dominique, alors qu'il travaillait avec Philippe Bergeron sur le film « Tony de Peltrie » dans un « garde-robe » situé juste à côté de la salle des terminaux VT200.

> On a un livre analogue avec Python qui s'en vient sous peu :-) 

Très bien!

> Et un autre en apprentissage machine pas mal avancé. Je suis tombé sur ton cours en vision en regardant ce qui a été fait dans le domaine en français et c'est du beau boulot ! Bonne année à toi aussi.

Il manque de bon matériel en français pour enseigner l'informatique et l'IA / science des données.

À la prochaine

Claude

> 
> Robert Godin

**Comment by lemire** [comment]
Le livre de Python s'en vient...

**Comment by RobertGodin** [comment]
Plutôt triste en effet... Petite anecdote avec Philippe Bergeron, on était dans le même garde-robe  si tu fais référence à un petit local avec deux terminaux si ma mémoire est bonne ? Je pense qu'à deux on bouffait 95% du CPU à un moment donné. Moi je roulais des jobs de construction de treillis qui pouvait prendre jusqu'à une semaine pour produire le résultat :-) Ils ont pris un certain temps avant de débusquer les coupables qui paralisait la machine !!!

**Comment by ClaudeCoulombe** [comment]
Remoi... 

Ce que j'apprécie particulièrement de votre livre que j'ai parcouru:

-  le chapitre 1 sur les concepts de base, pas très « sexés » par définition, mais essentiels à un débutant pour comprendre qu'un ordinateur et l'informatique ne sont pas magiques. Le contenu est de base mais sur la coche, y compris une initiation en douce à UML. J'ai appris que les mots coquille (shell) et noyau (kernel) venaient de l'analogie avec une noix! 

- l'utilisation des classes graphiques 2D et le projet de jeu interactif qui permet d'illustrer des concepts importants de la programmation par objets, les déclarations de types foisonnantes de Java, de la programmation par événements, de la programmation d'animations et surtout de motiver les apprenants.

Par contre, je trouve qu'il manque une petite touche artistique à vos robots et bonhommes allumettes... 
Rien n'est parfait, mais tout s'améliore.

 Claude

**Comment by RobertGodin** [comment]
Merci pour les commentaires ! Pour les bonhommes, l'objectif était un exemple minimaliste :-) On va essayer d'améliorer ça.
